python/pyspark/ml/image.py

```diff
@@ -75,6 +76,23 @@ def ocvTypes(self):
             self._ocvTypes = dict(ctx._jvm.org.apache.spark.ml.image.ImageSchema.javaOcvTypes())
         return self._ocvTypes
 
+    @property
+    def columnSchema(self):
+        """
+        Returns the schema for the image column.
+
+        :return: a :class:`StructType` for image column,
+            ``struct<origin:string, height:int, width:int, nChannels:int, mode:int, data:binary>``.
+
+        .. versionadded:: 2.3.0
```

Comment:
I am fine with 2.4.0. Let me know.

Reply:
Since this came out of the 2.3 ml QA and it's mostly an improvement to the python API, I think maybe 2.4 is best. But it is a new API do maybe it's ok to include in 2.3..

Reply:
Yea, let me go with 2.4.0.